generic/tests/fillup_disk.py

```diff
@@ -1,15 +1,52 @@
 import logging
+import re
 
 from virttest import error_context
 
 
+def fill_partitions(test, params, session, partition, deeply=False):
+    """
+    Fill up the specified partitions.
+    :param test: QEMU test object.
+    :param params: Dictionary with the test parameters.
+    :param session: ShellSession object.
+    :param partition: Specified partition,
+    :param deeply: fill up partition with block size of 512b.
+    """
+    fillup_cmd = params.get("fillup_cmd")
+    fillup_timeout = int(params.get("fillup_timeout"))
+    fillup_size = int(params.get("fillup_size"))
+    number = 0
+
+    info = "Start shallow filling the %s partition." % partition
+    if deeply:
+        info = "Start deep filling the %s partition with 512b." % partition
+    error_context.context(info, logging.info)
+
+    while 1:
+        # As we want to test the backing file, so bypass the cache
+        cmd = fillup_cmd % (partition, number, fillup_size)
+        if deeply:
+            cmd = 'dd if=/dev/zero of=%s/fillup.512b.%d bs=512b ' \
+                     'count=1 oflag=direct' % (partition, number)
```

Comment:
Suggest remove 'count=1', use the `dd` to fill the partition up.

Reply:
Good Suggestions, thanks.